README.md

```diff
@@ -1,19 +1,98 @@
+HydroShare is a collaborative website being developed for better access to data and models in the hydrologic sciences.
 
-Build generate by [Jenkins CI](http://ci.hydroshare.org:8080)
+#### Nightly Build Status generated by [Jenkins CI](http://ci.hydroshare.org:8080) (develop branch)
 
-### requires.io
-[![Requirements Status](https://requires.io/github/hydroshare/hs_docker_base/requirements.svg?branch=develop)](https://requires.io/github/hydroshare/hs_docker_base/requirements/?branch=master)
-
-Hydroshare
-============
+| Workflow | Clean | Build/Deploy | Unit Tests | Flake8 | Requirements |
+| -------- | ----- | ------------ | ---------- | -------| ------------ |
+| [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-workflow/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-workflow/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-clean/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-clean/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-deploy/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-deploy/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-test/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-test/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-flake8/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-flake8/) | [![Requirements Status](https://requires.io/github/hydroshare/hs_docker_base/requirements.svg?branch=develop)](https://requires.io/github/hydroshare/hs_docker_base/requirements/?branch=master) | 
 
 HydroShare is a collaborative website being developed for better access to data and models in the hydrologic sciences. HydroShare will provide the sustainable technology infrastructure needed to address critical issues related to water quantity, quality, accessibility, and management. HydroShare will expand the data sharing capability of the CUAHSI Hydrologic Information System by broadening the classes of data accommodated, expanding capability to include the sharing of models and model components, and taking advantage of emerging social media functionality to enhance information about and collaboration around hydrologic data and models. 
```

Comment:
Again - only hydrologists?  I suggest "...aimed at providing the cyberinfrastructure needed to enable innovation and collaboration in research to solve water problems."